x/perpetual/migrations/v8_migration.go

```diff
@@ -0,0 +1,43 @@
+package migrations
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/elys-network/elys/x/perpetual/types"
+)
+
+func (m Migrator) V8Migration(ctx sdk.Context) error {
+	legacyParams := m.keeper.GetLeagcyParams(ctx)
+	params := types.NewParams()
+
+	params.LeverageMax = legacyParams.LeverageMax
+	params.FundingFeeCollectionAddress = legacyParams.FundingFeeCollectionAddress
+	params.FundingFeeMinRate = legacyParams.FundingFeeMinRate
+	params.FundingFeeMaxRate = legacyParams.FundingFeeMaxRate
+	params.FundingFeeBaseRate = legacyParams.FundingFeeBaseRate
+	params.TakeProfitBorrowInterestRateMin = legacyParams.TakeProfitBorrowInterestRateMin
+	params.BorrowInterestRateDecrease = legacyParams.BorrowInterestRateDecrease
+	params.BorrowInterestRateIncrease = legacyParams.BorrowInterestRateIncrease
+	params.BorrowInterestRateMax = legacyParams.BorrowInterestRateMax
+	params.BorrowInterestRateMin = legacyParams.BorrowInterestRateMin
+	params.MinBorrowInterestAmount = types.NewParams().MinBorrowInterestAmount
+	params.EpochLength = legacyParams.EpochLength
+	params.ForceCloseFundAddress = legacyParams.ForceCloseFundAddress
+	params.ForceCloseFundPercentage = legacyParams.ForceCloseFundPercentage
+	params.HealthGainFactor = legacyParams.HealthGainFactor
+	params.IncrementalBorrowInterestPaymentEnabled = legacyParams.IncrementalBorrowInterestPaymentEnabled
+	params.IncrementalBorrowInterestPaymentFundAddress = legacyParams.IncrementalBorrowInterestPaymentFundAddress
+	params.IncrementalBorrowInterestPaymentFundPercentage = legacyParams.IncrementalBorrowInterestPaymentFundPercentage
+	params.InvariantCheckEpoch = legacyParams.InvariantCheckEpoch
+	params.LeverageMax = legacyParams.LeverageMax
+	params.MaxOpenPositions = legacyParams.MaxOpenPositions
+	params.PoolOpenThreshold = legacyParams.PoolOpenThreshold
+	params.SafetyFactor = legacyParams.SafetyFactor
+	params.WhitelistingEnabled = legacyParams.WhitelistingEnabled
```

Comment:
@avkr003 I noticed you introduce a new max limit order param, but it does not look like it is set here

Reply:
It's a PR merged from @cryptokage1996 https://github.com/elys-network/elys/pull/818
Since both PR changes perpetual PRs, combined into one.

Reply:
@avkr003 got it thanks for clarity, @cryptokage1996 can you update the PR to initiate the new parameter value within the migrator function, thanks

Reply:
@cryptokage1996 @avkr003 I pushed some changes to fix this